src/sentry/constants.py

```diff
@@ -189,6 +189,8 @@ def get_all_languages():
 
 OK_PLUGIN_SAVED = _('Configuration for the {name} integration has been saved.')
 
+WARN_SESSION_EXPIRED = 'Your session has expired.'  # TODO: translate this
```

Comment:
If you know a way to fix that, I could roll into this PR ... or we can just move forward and land this and follow-up. I'm in favor of the latter.